package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "license": "MIT",
+  "dependencies": {
+    "ajv-cli": "^5.0.0"
+  },
+  "scripts": {
+    "lint": "yarn lint-schema && yarn lint-uuids",
+    "lint-schema": "ajv validate -s config.json.schema.json -d config.json",
+    "lint-uuids": "bin/lint_uuids.sh"
```

Comment:
One cannot express the UUID's uniqueness property in JSON schema (at least not that I could find)